python/lsst/ts/externalscripts/maintel/tma/short_long_slews.py

```diff
@@ -0,0 +1,324 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["ShortLongSlews"]
+
+import asyncio
+
+import yaml
+from lsst.ts.idl.enums.Script import ScriptState
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+from lsst.ts.salobj import type_hints
+from lsst.ts.standardscripts.base_block_script import BaseBlockScript
+
+
+class ShortLongSlews(BaseBlockScript):
+    """Execute short and long slews for a grid of azimuths and elevations.
+
+    Overview:
+
+    This script performs a series of point-to-point (P2P) telescope
+    movements forming a diamond pattern around each user-defined grid
+    position with short and long slews. It is designed for dynamic tests
+    and performance evaluations, reproducing patterns used in specific
+    engineering tasks (e.g., BLOCK-T293 and T294).
+
+    Execution Order:
```

Comment:
The section mentions the `grid_az` followed by the `grid_el`. But it does not mention the diamond-shaped movements mentioned above.